### DISCUSSION TITLE
New Setup on L25 Probat. New to Artisan

### DISCUSSION THREAD

**Discussion by Dgocof** [artisan-roaster-scope/artisan#1009] [General]
I'm wanting to setup Artisan on a Probat L25 Natural Gas. I'm wondering which controller to go with for PID control for the modulating gas valve? Omron, Watlow, Fuji or other. I'm a coffee roaster not a techi so I will probably stay away from the Phidgets and arduino's. Which would give the best control and least complicated to setup? I'll probably set this up on a mac. Would you recommend purchasing the RS485 cable from the same company or a generic. There seems to be a considerable price difference.
I'll probably use a 2mm J thermocouple. Any feedback would be appreciated.

**Comment by bigl00z3** [comment]
I don't have experience on a gas valves. However for the PID I have had good experience with the Fuji PXF (and bad one with MaxWell).

Not on gas, but you can still check the Fuji related posts by Frans on [his website](https://kostverlorenvaart.blogspot.com/search?q=fuji) or the ones I did on mine [here](https://www.bigloose.com/v11/index.php/home/archive/artisan-roasting-with-the-pid-fuji-pxf-and-test-of-maxwell-mta) and [here](https://www.bigloose.com/v11/index.php/home/archive/artisan-roasting-with-the-pid-using-bt-as-input-fuji-pxf). Also the related Fuji discussions on this list [here](https://github.com/artisan-roaster-scope/artisan/discussions/763).

For the RS485 to USB, no need for the same company. You can check [here](https://github.com/artisan-roaster-scope/artisan/discussions/1000#discussioncomment-4033697) for my previous recommendations based on my usage. Many other models will also work.

Nico
